DQM/SiTrackerPhase2/plugins/Phase2TrackerMonitorDigi.cc

```diff
@@ -441,14 +441,14 @@ void Phase2TrackerMonitorDigi::bookLayerHistos(DQMStore::IBooker& ibooker,
     std::string top_folder = config_.getParameter<std::string>("TopFolderName");
     std::stringstream folder_name;
 
-    bool forDisc12UptoRing10 =
+    //For endCap: P-type sensors are present only upto ring 10 for discs 1&2 (TEDD-1) and upto ring 7 for discs 3,4&5 (TEDD-2)
```

Comment:
tagging it here in a semi-random place.
The static analyzer has found issue in this file (access to the ES without esConsumes): https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-95ab08/10783/llvm-analysis/
would you mind fixing that one too?